src/components/SwitchToggle.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import styled from 'styled-components/macro';
+import PropTypes from 'prop-types';
+
+const Checkbox = styled.input.attrs({ type: 'checkbox' })`
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+`;
+
+const Label = styled.label`
+  align-items: center;
+  display: flex;
+  min-height: 2rem;
+  padding-left: 3.5rem;
+  position: relative;
+  user-select: none;
+  &::before {
+    background-color: ${theme => theme.input};
+    border: 1px solid
+      ${({ checked, theme }) => (checked ? theme.border : '#ece9f1')};
+    border-radius: 100rem;
+    box-shadow: ${({ theme }) => theme.boxShadowSmall};
```

Comment:
`theme.boxShadowSmall` is not defined in `themes.js`. You should add `boxShadowSmall: '0 6px 24px rgba(0, 0, 0, 0.07)',` after `boxShadowHeader.